exercises/pangram/src/main/java/Pangrams.java

```diff
@@ -0,0 +1,6 @@
+public class Pangrams {
```

Comment:
Thanks for talking that through with me, Frida.  That's solid thinking; I'm totally on the same page with you.

I also like the idea of renaming the class as you mentioned.